infra/reachability-metadata/src/main/resources/META-INF/native-image/org.apache.shardingsphere/generated-reachability-metadata/jni-config.json

```diff
@@ -0,0 +1,2 @@
+[
+]
```

Comment:
- @terrymanu As you can see, the new JSON entries generated by `./mvnw -PgenerateMetadata -DskipNativeTests -e -T1C clean test native:metadata-copy` do not carry the empty line at the end of the file, and the checkstyle plugin does not check the JSON . Is it necessary to add a blank line at the end of the file?

Reply:
It is fine